salt/states/dockerng.py

```diff
@@ -1389,11 +1389,20 @@ def running(name,
         # Add any needed container creation arguments based on the validated
         # runtime arguments.
         if runtime_kwargs.get('binds') is not None:
-            # Using bind mounts requries mountpoints to be specified at the
-            # time the container is created, so we need to add them to the
-            # create_kwargs.
-            create_kwargs['volumes'] = [bind_conf['bind'] for bind_conf in
-                                        runtime_kwargs['binds'].values()]
+            if 'volumes' not in create_kwargs:
+                # Check if there are preconfigured volumes in the image
+                preconfigured_volumes = False
+                for step in __salt__['dockerng.history'](image, quiet=True):
```

Comment:
you can get rid of 
`preconfigured_volumes` variable
with the `for` `else` pattern.
```python
for i in iter:
    if condition:
        break
else:
    # enter here only if the loop was entirely consumed (not breaked)
```